### PR TITLE
Add Crn attribute

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_workspace.go [hunk 1]
```diff
@@ -121,6 +122,11 @@ func resourceIBMPIWorkspaceRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 	d.Set(PIWorkspaceName, controller.Name)
+	wsDetails := map[string]interface{}{
+		WorkspaceCreationDate: controller.CreatedAt,
+		WorkspaceCRN:          controller.TargetCRN,
+	}
+	d.Set(Attr_WorkspaceDetails, flex.Flatten(wsDetails))
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_workspace.go:129]: I do not see `Attr_WorkspaceDetails` defined in the resource schema. Did you run the tests to ensure the code works?
Also, please update the docs with the new attribute.

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: ibm/service/power/resource_ibm_pi_workspace.go (hunk 1 above)
Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_workspace.go:129]: Added

File: ibm/service/power/resource_ibm_pi_workspace.go [hunk 2]
```diff
@@ -121,6 +122,11 @@ func resourceIBMPIWorkspaceRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 	d.Set(PIWorkspaceName, controller.Name)
+	wsDetails := map[string]interface{}{
+		WorkspaceCreationDate: controller.CreatedAt,
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_workspace.go:126]: Could add this in the doc as well?

**Comment by ismirlia** [comment]
Sorry I accidentally pushed to this branch thinking I was committing to a different repo. This should be fixed now.

File: ibm/service/power/resource_ibm_pi_workspace.go (hunk 2 above)
Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_workspace.go]: Added

File: website/docs/r/pi_workspace.html.markdown [hunk 3]
```diff
@@ -1,49 +1,50 @@
...
+## Argument reference
+
+Review the argument references that you can specify for your resource.
+
+- `pi_name` - (Required, String) A descriptive name used to identify the workspace.
+- `pi_datacenter` - (Required, String) Target location or environment to create the resource instance.
+- `pi_resource_group_id` - (Required, String) The ID of the resource group where you want to create the workspace. You can retrieve the value from data source `ibm_resource_group`.
+- `pi_plan` -  (Required, String) Plan associated with the offering; Valid values are `public` or `private`.
+- `crn` - (String) Workspace crn
+- `creation_date` -(String) Workspace creation Date
```
Comment (michaelkad) [inline website/docs/r/pi_workspace.html.markdown:45]: Those two are not `Argument`. There should be under `pi_workspace_details` in the `Attribute` section

**Review by michaelkad — CHANGES_REQUESTED** [review]

File: ibm/service/power/resource_ibm_pi_workspace.go (hunk 1 above)
Comment (yussufsh) [reply ibm/service/power/resource_ibm_pi_workspace.go:129]: I still don't see it in the schema.

File: website/docs/r/pi_workspace.html.markdown [hunk 4]
```diff
@@ -1,49 +1,51 @@
...
-- `pi_resource_group_id` - (Required, String) The ID of the resource group where you want to create the workspace. You can retrieve the value from data source `ibm_resource_group`.
-- `pi_plan` -  (Required, String) Plan associated with the offering; Valid values are `public` or `private`.
-
-## Attribute reference
-
-In addition to all argument reference listed, you can access the following attribute references after your resource source is created.
-
-- `id` - (String) Workspace ID.
\ No newline at end of file
+---
```
Comment (yussufsh) [inline website/docs/r/pi_workspace.html.markdown:1]: Cannot make out what's changed in this file.

Comment (Diptipowervs) [reply website/docs/r/pi_workspace.html.markdown:1]: - `crn` - (String) Workspace crn
- `creation_date` -(String) Workspace creation Date Attributes added at the end of the document

File: ibm/service/power/resource_ibm_pi_workspace.go (hunk 1 above)
Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_workspace.go:135]: Added in schema

**Comment by Diptipowervs** [comment]
Test Results:--- PASS: TestAccIBMPIWorkspaceBasic (144.96s)
PASS

File: ibm/service/power/resource_ibm_pi_workspace_test.go [hunk 5]
```diff
@@ -37,9 +37,9 @@ func testAccCheckIBMPIWorkspaceConfig(name string) string {
 	return fmt.Sprintf(`
 	 resource "ibm_pi_workspace" "powervs_service_instance" {
 		pi_name              = "%[1]s"
-		pi_datacenter        = "dal"
+		pi_datacenter        = "satloc_dal_clp25g92006uiuvbvku0"
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_workspace_test.go:40]:
```suggestion
		pi_datacenter        = "dal"
```

File: website/docs/r/pi_workspace.html.markdown [hunk 6]
```diff
@@ -1,49 +1,51 @@
...
+- `pi_resource_group_id` - (Required, String) The ID of the resource group where you want to create the workspace. You can retrieve the value from data source `ibm_resource_group`.
+- `pi_plan` -  (Required, String) Plan associated with the offering; Valid values are `public` or `private`.
+
+## Attribute reference
+
+In addition to all argument reference listed, you can access the following attribute references after your resource source is created.
+
+- `id` - (String) Workspace ID.
+- `crn` - (String) Workspace crn
+- `creation_date` -(String) Workspace creation Date
```
Comment (michaelkad) [inline website/docs/r/pi_workspace.html.markdown:51]:
```suggestion
- `workspace_details` - (Map) Workspace information.

    Nested schema for `pi_workspace_details`:
  - `creation_date` - (String) Date of workspace creation.
  - `crn` - (String) Workspace crn.
```

**Review by michaelkad — COMMENTED** [review]
Documentation needs update.

**Review by yussufsh — APPROVED** [review]

**Review by michaelkad — APPROVED** [review]

**Pull request merged**